lib/utils/copy-utils.js

```diff
@@ -34,14 +34,14 @@ var generatorCopy = (
  * the template files.
  * @returns {Function} A curried function that takes a file path and copies it
  */
-var generatorCopyTpl = (
+const generatorCopyTpl = (
 	generator,
 	templateDir,
 	templateData
 ) => /** @param {string} filePath */ filePath => {
-	var sourceParts = templateDir.split(path.delimiter);
+	const sourceParts = templateDir.split(path.delimiter);
 	sourceParts.push.apply(sourceParts, filePath.split("/"));
```

Comment:
Mutable object, but okay..

Reply:
It doesn't get reassigned, so `const` is ok. Otherwise eslint would have failed :)

Reply:
The `.push` method is mutating it, ( even though the push method is immutable ) but that's another thing :D